src/Common/src/System/Net/ContextAwareResult.cs

```diff
@@ -180,9 +194,13 @@ internal WindowsIdentity Identity
         {
             get
             {
-                GlobalLog.Assert(!InternalPeekCompleted || (_flags & StateFlags.ThreadSafeContextCopy) != 0, "ContextAwareResult#{0}::Identity|Called on completed result.", LoggingHash.HashString(this));
                 if (InternalPeekCompleted)
                 {
+                    if ((_flags & StateFlags.ThreadSafeContextCopy) == 0)
+                    {
+                        GlobalLog.AssertFormat("ContextAwareResult#{0}::Identity|Called on completed result.", LoggingHash.HashString(this));
+                    }
```

Comment:
Why is this call to `GlobalLog.AssertFormat` not protected by a call to `Globallog.IsEnabled` but other calls to `GlobalLog.AssertFormat` are protected?